code/modules/cm_marines/shuttle_backend.dm

```diff
@@ -407,8 +407,8 @@ cdel(src)
 		y = C.y_pos
 		C1.x_pos = x*cos(deg) + y*sin(deg)
 		C1.y_pos = y*cos(deg) - x*sin(deg)
-		C1.x_pos = RoundNearest(C.x_pos) //Sometimes you get very close to the right number but off by around 1e-15 and I want integers dammit
-		C1.y_pos = RoundNearest(C.y_pos)
+		C1.x_pos = round(C.x_pos) //Sometimes you get very close to the right number but off by around 1e-15 and I want integers dammit
+		C1.y_pos = round(C.y_pos)
```

Comment:
round without a second argument acts like floor which is probably why some mistaken individual made RoundNearest. `round(x, 1)` is the proper replacement